src/qt/sendcoinsdialog.cpp

```diff
@@ -908,11 +950,11 @@ void SendConfirmationDialog::updateYesButton()
     if(secDelay > 0)
     {
         yesButton->setEnabled(false);
-        yesButton->setText(tr("Send") + " (" + QString::number(secDelay) + ")");
+        yesButton->setText(confirmButtonText + " (" + QString::number(secDelay) + ")");
```

Comment:
Suggest using 4e10bdd433f5d1ba9069f4942cb89e69dd83d816 for this

Reply:
I'll rebase if that's merged first, but that's not a trivial enough commit.